client/components/ProjectView.tsx

```diff
@@ -103,7 +103,14 @@ export default function ProjectView({ selectedProject, projectLoading }: Project
             <div id="dashWrapper">
                 <div className="loggerBox" />
                 <div className="chartBox">
-                    {projectLoading ? <Loading /> : <div id="loading"> Select a project</div>}
+                    {projectLoading ? (
+                        <Loading />
+                    ) : (
+                        <div id="loading">
+                            {' '}
```

Comment:
This spaces doesn't seem to serve a purpose. Everything looked ok on my end if I removed it

Reply:
Yup

Reply:
To be addressed in #86